Kentico.Kontent.Management/Modules/ActionInvoker/ImageTransformationConverter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Kentico.Kontent.Management.Models.AssetRenditions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Kentico.Kontent.Management.Modules.ActionInvoker;
+
+internal class ImageTransformationConverter : JsonConverter
+{
+    private static readonly JsonSerializerSettings _specifiedSubclassConversion = new JsonSerializerSettings() { ContractResolver = new BaseSpecifiedConcreteClassConverter() };
+
+    public override bool CanConvert(Type objectType) => objectType == typeof(ImageTransformation);
+
+    public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+    {
+        var jObject = JObject.Load(reader);
+        var type = jObject["mode"].ToObject<ImageTransformationMode>();
```

Comment:
Suggestion: Possible null reference exception, some nice and descriptive exception could be thrown.

Reply:
Hmm, I pretty much copied the code from `ElementMetadataConverter`. Anyway, null handling + custom exception added. Let me know if I should have picked some other exception/better message.